### PR TITLE
[bitnami/*] Standardize documentation

### PR DESCRIPTION
**Pull request by jotamartos**
### Description of the change

Standardize documentation for all charts in the catalog

### Benefits

Documentation can be reused in all projects. It's not dependent of the distro or repository information.  

### Checklist

- [NA] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [NA] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [X] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/discourse/README.md [hunk 1]
```diff
@@ -36,9 +36,11 @@ Looking to use Discoursereg; in production? Try [VMware Application Catalog](htt
 To install the chart with the release name `my-release`:
 
 ```console
-helm install my-release oci://registry-1.docker.io/bitnamicharts/discourse
+helm install my-release oci://registry_name/repository_name/discourse
 ```
 
+> Please replace the registry_name and repository_name placeholders with a reference to your chart registry & repository.
```
Comment (carrodher) [inline bitnami/discourse/README.md:42]:
```suggestion
> Note: you need to substitute the placeholders REGISTRY_NAME and REPOSITORY_NAME with a reference to your Helm chart registry and repository.
```

According to other examples of placeholders, they are defined in uppercase and with a similar wording as the suggested one

File: bitnami/discourse/README.md [hunk 2]
```diff
@@ -36,9 +36,11 @@ Looking to use Discoursereg; in production? Try [VMware Application Catalog](htt
 To install the chart with the release name `my-release`:
 
 ```console
-helm install my-release oci://registry-1.docker.io/bitnamicharts/discourse
+helm install my-release oci://registry_name/repository_name/discourse
```
Comment (carrodher) [inline bitnami/discourse/README.md:39]:
```suggestion
helm install my-release oci://REGISTRY_NAME/REPOSITORY_NAME/discourse
```

According to other examples of placeholders, they are defined in uppercase.

**Review by carrodher — COMMENTED** [review]
Thanks for the changes, in general, they LGTM. Just a comment that could be applied to the rest of sections

File: bitnami/discourse/README.md [hunk 3]
```diff
@@ -332,19 +332,23 @@ Specify each parameter using the `--set key=value[,key=value]` argument to `helm
 ```console
 helm install my-release \
   --set auth.username=admin,auth.password=password \
-    oci://registry-1.docker.io/bitnamicharts/discourse
+    oci://registry_name/repository_name/discourse
```
Comment (raquel-campuzano) [inline bitnami/discourse/README.md:335]: Please apply the comment above related to uppercase placeholders.

File: bitnami/discourse/README.md [hunk 4]
```diff
@@ -332,19 +332,23 @@ Specify each parameter using the `--set key=value[,key=value]` argument to `helm
...
 ```
 
+> Please replace the registry_name and repository_name placeholders with a reference to your chart registry & repository.
```
Comment (raquel-campuzano) [inline bitnami/discourse/README.md:338]: Reword the note according to the uppercase placeholder rule.

**Comment by raquel-campuzano** [comment]
Please apply the comments related to placeholders for consistency with the rest of documentation. Thank you!

**Comment by jotamartos** [comment]
Waiting for https://github.com/bitnami-labs/readme-generator-for-helm/pull/78 before updating all README files in this PR.

File: .github/workflows/generate-chart-readme.yml [hunk 5]
```diff
@@ -35,7 +35,7 @@ jobs:
           curl -Lkso $TEMP_FILE $DIFF_URL
           files_changed="$(sed -nr 's/[\-\+]{3} [ab]\/(.*)/\1/p' $TEMP_FILE | sort | uniq)"
           # Adding || true to avoid "Process exited with code 1" errors
-          charts_dirs_changed="$(echo "$files_changed" | xargs dirname | grep -o "bitnami/[^/]*" | sort | uniq || true)"
+          charts_dirs_changed="$(echo "$files_changed" | xargs dirname | grep -o "bitnami/[^/]*" | grep -v "bitnami/common" | sort | uniq || true)"
```
Comment (jotamartos) [inline .github/workflows/generate-chart-readme.yml:38]: Please review this @fmulero @carrodher.

File: README.md [hunk 6]
```diff
@@ -10,9 +10,11 @@ Looking to use our applications in production? Try [VMware Application Catalog](
 ## TL;DR
 
 ```bash
-helm install my-release oci://registry-1.docker.io/bitnamicharts/<chart>
+helm install my-release oci://REGISTRY_NAME/REPOSITORY_NAME/<chart>
 ```
 
+> Note: You need to substitute the placeholders REGISTRY_NAME and REPOSITORY_NAME with a reference to your Helm chart registry and repository. For example, in the case of Bitnami, you need to use REGISTRY_NAME=registry-1.docker.io and REPOSITORY_NAME=bitnamicharts.
```
[the remark this answers is not in the document]
Comment (jotamartos) [reply README.md:16]: This info looks better this way. I just applied the changes.

**Review by carrodher — DISMISSED** [review]

**Review by fmulero — APPROVED** [review]
LGTM

**Review by fmulero — APPROVED** [review]
